.gitignore

```diff
@@ -45,6 +45,7 @@ ipch/
 *.VC.opendb
 .vs/
 .vscode/
+.idea/
```

Comment:
Please don't change `.gitignore`. There is another PR specifically for this and it is still being discussed.